backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientResolver.java

```diff
@@ -20,12 +20,22 @@
     @Autowired
     private PersonService ps;
 
-    public List<Person> getPatients(UUID facilityId) {
-        return ps.getPatients(facilityId);
+    public List<Person> getPatients(UUID facilityId, long pageNumber, long pageSize, boolean showDeleted) {
+        if (pageNumber < 0) {
+            pageNumber = PersonService.DEFAULT_PAGINATION_PAGEOFFSET;
+        }
+        if (pageSize < 1) {
+            pageSize = PersonService.DEFAULT_PAGINATION_PAGESIZE;
+        }
+        if (facilityId == null) {
+            return showDeleted ? ps.getAllPatientsInclDeleted() : ps.getAllPatients();
```

Comment:
This seems different then what I would expect from the mocs. In the mock attached to https://app.zenhub.com/workspaces/simplereport-5ff8be8854ae8b000e378210/issues/cdcgov/prime-simplereport/612 we are adding two tabs to the people tab one with all active people and one with only archived people

I would expect this to be
`return showDeleted ? ps.getAllArchivedPatients() : ps.getAllPatients();`

Reply:
The issue I see with the `getAllPatientsInclDeleted` approach is to meet the mocks the frontend will have to implement the filtering out of people which will break pagination. You could have the case where the user is viewing the archived persons tab. requests a new page and recives a blank page because all the records are not archived.

Reply:
@dmethvin-gov 
Good point about the pagination becomes an issue if we filter just on the client. 

Should we change it from `showDeleted` to `showOnlyDeleted`?

Reply:
Yeah, to be able to do V2 with a separate list and also paginate it looks like we'd need to have a way to only show the archived records. So it sounds like there is some more work to do.

Reply:
I "fixed" this. The server side calls now behave as `getAllArchivedPatients()` and `getAllPatients()` The front-end code compiles and runs but I doubt it's doing the right thing. 

@dmethvin-gov did you want to work off this branch and implement v2?

Reply:
Update: we're deferring the "show archived patients in a separate tab". Archived patients will just disappear for now.